ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeBloomer.java

```diff
@@ -229,7 +228,7 @@ private boolean exceedMaxNumberOfRaPerTso(NetworkActionCombination naCombination
 
         // get max number of network action which can still be activated, per Tso
         tsos.forEach(tso -> {
-            int activatedPstsForTso = (int) fromLeaf.getRangeActions().stream().filter(rangeAction -> rangeAction instanceof PstRangeAction && hasRangeActionChangedComparedToPrePerimeter(fromLeaf, rangeAction)).count();
+            int activatedPstsForTso = (int) fromLeaf.getRangeActions().stream().filter(rangeAction -> hasRangeActionChangedComparedToPrePerimeter(fromLeaf, rangeAction)).count();
```

Comment:
rename this to activatedRangeActionsForTso

Reply:
ok